boxoffice/extapi/stats.py

```diff
@@ -0,0 +1,28 @@
+from boxoffice import db, app
+from boxoffice.models import ItemCollection, LineItem, Order, ORDER_STATUS
+from datetime import datetime as dt
+import requests
+from urllib import urlencode
+import json
+from tabulate import tabulate
+from rq.decorators import job
+
+@job('boxoffice')
+def post_stats(id, room=None):
+    if id is None:
+        return None
+
+    with app.test_request_context():
+        if room is None:
+            room = '#test'
```

Comment:
Do we need to specify a default channel? Does slack pick a default channel automatically by any chance?